barretenberg/cpp/src/barretenberg/dsl/acir_proofs/acir_composer.cpp

```diff
@@ -17,6 +17,60 @@ AcirComposer::AcirComposer(size_t size_hint, bool verbose)
     , verbose_(verbose)
 {}
 
+/**
+ * @brief Splits a vector into two vectors,
+ * the first containing the first 32 * k elements, and the second containing
+ * the rest.
+ *
+ * @param original - The original vector to split
+ * @param k - The number of 32 bytes to remove
+ * @return std::pair<std::vector<uint8_t>, std::vector<uint8_t>>
+ */
+std::pair<std::vector<uint8_t>, std::vector<uint8_t>> splitVector(std::vector<uint8_t>& original, uint32_t k)
```

Comment:
We already have some utility functions for this kind of thing:
barretenberg/cpp/src/barretenberg/common/container.hpp